README.md

```diff
@@ -43,7 +43,35 @@ _Note:_ You have to have the android simulator running before running `react-nat
 
 ### On a Device
 
-__HELP REQUESTED__: I don't have an Android device so I can't test this out.
+#### For Linux
+##### Enable USB Debugging
```

Comment:
USB debugging is need for linux or mac, could you move this out of the "For Linux" section and just into the "On a Device" section?